bbp/modules/default.nix

```diff
@@ -1597,8 +1597,8 @@ let
                pythonPkgs.six
                pythonPkgs.tables
             ];
-            conflicts = [ python27-light python36-full python36-light ] ++ conflicts-modules;
-            dependencies = [ nss-wrapper gcc manylinux1-python ];
+            conflicts = [ python36-light python36-full ] ++ conflicts-modules;
+            dependencies = [ python27-light gcc ];
```

Comment:
should we move `gcc` dependency to `python27-light`?

Reply:
should we remove the packages already specified by `python27-light` from this list?